config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -42,7 +42,7 @@ presubmits:
         - --provider=gce
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-storage-slow
         - --test_args=--ginkgo.focus=\[sig-storage\].*\[Slow\] --ginkgo.skip=\[Driver:.gcepd\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
-        - --timeout=80m
+        - --timeout=120m
```

Comment:
I think you also need to increase the timeout at L33

Reply:
Done, thanks!